controller/tests/array_action/svc/array_mediator_svc_test.py

```diff
@@ -408,21 +408,28 @@ def test_create_snapshot_start_fcmap_error(self, mock_warning):
             CLIFailureError("Failed")]
 
         with self.assertRaises(CLIFailureError):
-            self.svc.create_snapshot("test_snapshot", "source_volume")
+            self.svc.create_snapshot("test_snapshot", "source_volume", "pool1")
 
     def test_create_snapshot_success(self):
         self._prepare_mocks_for_create_snapshot()
 
-        snapshot = self.svc.create_snapshot("test_snapshot", "source_volume")
+        snapshot = self.svc.create_snapshot("test_snapshot", "source_volume", "pool1")
 
         self.assertEqual(snapshot.capacity_bytes, 1024)
         self.assertEqual(snapshot.array_type, 'SVC')
         self.assertEqual(snapshot.id, 'snap_id')
 
+    def test_create_snapshot_with_different_pool_success(self):
+        self._prepare_mocks_for_create_snapshot()
+
+        self.svc.create_snapshot("test_snapshot", "source_volume", "different_pool")
```

Comment:
consider changing `"source_volume"` to something like `"source_volume_id"` wherever relevant

Reply:
done